recipes/probableparsing/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "probableparsing" %}
+{% set version = "0.0.1" %}
+{% set hashtype = "md5" %}
+{% set hash = "960a1fab176276070111f1f9b2080cca" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hashtype }}: {{ hash }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - future
+    run:
+        - python
+        - future
+test:
+    imports:
+        - {{name}}
```

Comment:
I added a PR ( https://github.com/conda-forge/probableparsing-feedstock/pull/1 ) to make this change.